File: src/programmers/level2/corrent_bracket_12909/solution.py
# def solution(s):
#     st = []
#     for i in range(len(s)):
#         if s[i] == '(':
#             st.append(i)
#         elif s[i] == ')':
#             if len(st) == 0:
#                 return False
#             st.pop()
#     if not len(st) == 0:
#         return False
#     return True


def solution(s):
    answer = True

    while s:
        if '()' in s :
            s = s.replace('()','')
        else:
            answer = False
            break

    return answer

File: src/programmers/level2/corrent_bracket_12909/Solution.java
package programmers.level2.corrent_bracket_12909;
import java.util.Stack;

class Solution {
    boolean solution(String s) {
        Stack<Integer> stack = new Stack<>();
        for(int i=0; i<s.length(); i++){
            if(s.charAt(i) == '('){
                stack.push(i);
            }else if(s.charAt(i) == ')'){
                if(stack.size() == 0){
                    return false;
                }
                stack.pop();
            }
        }

        if(stack.size() != 0){
            return false;
        }

        return true;
    }
}